### PR TITLE
DEV: flexify post-controls and stop relying on clearfix

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

**Comment by jjaffeux** [comment]
@awesomerobot @pmusaraj How do you feel about this ?

File: app/assets/stylesheets/desktop/topic-post.scss [hunk 1]
```diff
@@ -49,11 +49,16 @@ section.post-menu-area {
...
   padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 
   .actions {
     text-align: right;
     float: right;
     display: inline-block;
+    margin-left: auto;
```
Comment (pmusaraj) [inline app/assets/stylesheets/desktop/topic-post.scss:60]: Do you still need to `float: right`?

**Review by pmusaraj — APPROVED** [review]
Yeah, I'm all for this, we should be phasing out clearfix everywhere slowly.

File: app/assets/stylesheets/desktop/topic-post.scss (hunk 1 above)
Comment (jjaffeux) [reply app/assets/stylesheets/desktop/topic-post.scss:60]: no indeed :+1:

**Pull request merged**